documentation/server-reflection-tutorial.md

```diff
@@ -27,25 +27,23 @@ need to make the following changes:
    compile "io.grpc:grpc-protobuf:${grpcVersion}"
 +  compile "io.grpc:grpc-services:${grpcVersion}"
    compile "io.grpc:grpc-stub:${grpcVersion}"
-
+ 
    testCompile "junit:junit:4.11"
-diff --git a/examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java b/examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java
-index ce1158a4..788bcc62 100644
 --- a/examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java
 +++ b/examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java
 @@ -33,6 +33,7 @@ package io.grpc.examples.helloworld;
-
+ 
  import io.grpc.Server;
  import io.grpc.ServerBuilder;
-+import io.grpc.protobuf.service.ProtoReflectionService;
++import io.grpc.protobuf.services.ProtoReflectionService;
  import io.grpc.stub.StreamObserver;
-
  import java.io.IOException;
-@@ -51,6 +52,7 @@ public class HelloWorldServer {
+ import java.util.logging.Logger;
```

Comment:
This is pretty confusing in Github's diff view, but this is actually not a change in the file. This line is already present in the example servers, it's just new that it's showing in the diff window here after https://github.com/grpc/grpc-java/pull/2782 removed an empty line.

https://github.com/ericgribkoff/grpc-java/blob/c690686b12c64e1e9da2a809d47dbd5a5c30290c/documentation/server-reflection-tutorial.md is easier to parse.